content/academy/web_scraping_for_beginners.md

```diff
@@ -32,39 +32,40 @@ This is what you'll learn in the **Web scraping for beginners** course:
-  * [GraphQL Scraping]({{@link api_scraping/graphql_scraping.md}})
-    * [Introspection]({{@link api_scraping/graphql_scraping/introspection.md}})
-    * [Modifying variables]({{@link api_scraping/graphql_scraping/modifying_variables.md}})
-* [Anti-scraping protections]({{@link anti_scraping.md}})
-  * [Proxies]({{@link anti_scraping/mitigation/proxies.md}})
-  * Captchas and human behavior
-  * Web application firewalls
-* Apify Platform
-  * Getting started
-  * Apify CLI
-* Expert topics
-  * Mobile app scraping
-  * Overcoming result limits -->
-
 > We release course content as soon as we write it instead of launching it all at the same time much later. If you want to get notified about new lessons in the academy, [sign up for a free Apify account](https://console.apify.com/sign-up?asrc=developers_portal) to get our newsletters.
 
 ## [](#requirements) Requirements
 
-You don't need to be a developer or a software engineer to complete this course, but basic programming knowledge is recommended. Don't be afraid, though. We explain everything in great detail in the **Web scraping for beginners** course and provide external references that can help you level up your web scraping and development skills. If you're new to programming, pay very close attention to the instructions and examples. A seemingly insignificant thing like using `[]` instead of `()` can make a lot of difference.
+You don't need to be a developer or a software engineer to complete this course, but basic programming knowledge is recommended. Don't be afraid, though. We explain everything in great detail in the this course and provide external references that can help you level up your web scraping and development skills. If you're new to programming, pay very close attention to the instructions and examples. A seemingly insignificant thing like using `[]` instead of `()` can make a lot of difference.
+
+> If you don't already have basic programming/web development knowledge and would like to be well-prepared for this course, we recommend taking a [JavaScript course](https://www.codecademy.com/learn/introduction-to-javascript) and learning about [CSS Selectors](https://www.w3schools.com/css/css_selectors.asp).
+
+As you progress to the more advanced courses, the coding will get more challenging, but will still be manageable to a person with an intermediate level of programming skills.
+
+Ideally, you should have at least a moderate understanding of the following concepts:
+
+### [](#javascript-and-node) JavaScript + Node.js
+
+It is recommended to understand at least the fundamentals of JavaScript and be proficient with Node.js prior to developing an actor on the Apify platform. If you are not yet comfortable with asynchronous programming (with promises and `async...await`), loops (and the different types of loops in JavaScript), modularity, or working with external packages, we would recommend studying the following resources before coming back and continuing this section:
```

Comment:
Please go through all the content and remove references to actors and Apify from the texts where they are irrelevant or don't make sense. This course is not about actors at all.

```suggestion
It is recommended to understand at least the fundamentals of JavaScript and be proficient with Node.js prior to starting this course. If you are not yet comfortable with asynchronous programming (with promises and `async...await`), loops (and the different types of loops in JavaScript), modularity, or working with external packages, we would recommend studying the following resources before coming back and continuing this section:
```